pinot-controller/src/main/resources/app/pages/SchemaPageDetails.tsx

```diff
@@ -204,143 +222,146 @@ const SchemaPageDetails = ({ match }: RouteComponentProps<Props>) => {
   const handleSegmentDialogHide = () => {
     setShowEditConfig(false);
     setReloadSegmentsOnUpdate(false);
-  }
+  };
 
-  return fetching ? (
-    <AppLoader />
-  ) : (
-    <Grid
-      item
-      xs
-      style={{
-        padding: 20,
-        backgroundColor: 'white',
-        maxHeight: 'calc(100vh - 70px)',
-        overflowY: 'auto',
-      }}
-    >
-      <div className={classes.operationDiv}>
-        <SimpleAccordion
-          headerTitle="Operations"
-          showSearchBox={false}
-        >
-          <div>
-      <CustomButton
-              onClick={()=>{
-                setActionType('editSchema');
-                setConfig(JSON.stringify(schemaJSON, null, 2));
-                setShowEditConfig(true);
-              }}
-              tooltipTitle="Edit Schema"
-              enableTooltip={true}
-            >
-              Edit Schema
-            </CustomButton>
-            <CustomButton
-              isDisabled={!schemaJSON} onClick={handleDeleteSchemaAction}
-              tooltipTitle="Delete Schema"
-              enableTooltip={true}
-            >
-              Delete Schema
-            </CustomButton>
+  if (fetching) {
+    return <AppLoader />;
+  } else if (schemaNotFound) {
+    return <NotFound message={`Schema ${schemaName} not found`} />;
+  } else {
+    return (
+      <Grid
+        item
```

Comment:
minor nit: if you are returning from if then it can just be 

```js
if(condition1) {
    return 1;
}
if(condition2) {
    return 2;
}
return default;
```

Reply:
i know there's some convo in JS around whether you should have `else` when all the other branches return since you might accidentally return nothing. But I will leave this as is for now since everything else is implemented like this.